utils.py

```diff
@@ -157,3 +159,22 @@ def balance(it):
 
                 last_label = e[1]
                 yield e
+
+
+def download(url, filename, block='█'):
```

Comment:
Let's remove the `block` parameter and hardcode the character directly in the function.